src/getbits.rs

```diff
@@ -197,4 +197,11 @@ impl<'a> GetBits<'a> {
     pub const fn has_pending_bits(&self) -> bool {
         self.state != 0 || self.bits_left != 0
     }
+
+    pub fn get_bytes(&mut self, n: usize) -> Box<[u8]> {
```

Comment:
```suggestion
    pub fn get_bytes(&mut self, n: usize) -> &[u8] {
```

Reply:
I think it's better to leave it up to the caller if they want to allocate or not.

Reply:
Agreed